sdk/monitor/monitor-opentelemetry/src/metrics/performanceCounters.ts

```diff
@@ -238,7 +239,11 @@ export class PerformanceCounterMetrics {
   }
 
   private getPrivateMemory(observableResult: ObservableResult) {
-    observableResult.observe(process.memoryUsage().rss);
+    if (process?.memoryUsage) {
+      observableResult.observe(process.memoryUsage().rss);
+    } else {
+      Logger.getInstance().warn("Couldn't report Private Memory. Process is not defined.");
```

Comment:
Question: Do these really need to be warnings?

ie. If the user is specifically opt-in, then I can see these being warnings, but if we are just "doing our best" to collect these, then these diagnostic logs are probably info or debug level.

Reply:
That's fair. I was just thinking if they're explicitly enabling performance counters they should be alerted as to why they're getting invalid data. With some further consideration I think it makes most sense to just make them debug so if customers want to investigate why they're not seeing these metrics they can, but we won't spam the console with warnings.